conf/monitor/monitor.yml

```diff
@@ -0,0 +1,29 @@
+http-server:
```

Comment:
The [conf/monitor/monitor.yaml](https://github.com/erda-project/erda/blob/develop/conf/monitor/monitor.yaml) already exists, and the two config files need to be merged.